libr/core/linux_heap_glibc.c

```diff
@@ -83,8 +83,12 @@ static void GH(print_main_arena)(RCore *core, GHT m_arena, GH(RHeap_MallocState)
 	PRINTF_BA ("0x%08x\n", (ut32)main_arena->flags);
 	PRINT_GA ("  fastbinsY = {\n");
 
-	for (i = 0, j = 1, k = SZ * 4; i < NFASTBINS - 2; i++, j++, k += SZ * 2) {
-		PRINTF_YA (" Fastbin %02d: ", j);
+	for (i = 0, j = 1, k = SZ * 4; i < NFASTBINS; i++, j++, k += SZ * 2) {
+		if (FASTBIN_IDX_TO_SIZE(j) <= global_max_fast) {
+			PRINTF_YA (" Fastbin %02d\n", j);
+		} else {
+			PRINTF_RA (" Fastbin %02d\n", j);
```

Comment:
I used this [macro](https://github.com/sferrini/radare2/blob/31ca4dee68bdb019283fd6920549759a9f759bf4/libr/include/r_heap_glibc.h#L20), doesn't it use the r2 color theme palette?